Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.h

```diff
@@ -36,6 +36,9 @@ namespace AzPhysics
         void SetMassComputeFlags(MassComputeFlags flags);
 
         bool IsCcdEnabled() const;
+        bool CcdReadOnly() const;
+        AZStd::string_view GetCcdTooltip() const;
+        AZStd::string_view GetKinematicTooltip() const;
```

Comment:
I saw @AMZN-daimini's earlier comments and as the strings are known at compile time (constant) then returning a `string_view` is safe but it's definitely something to be careful about because if this code ever changes there's the risk of a dangling reference. Unless it's performance critical return by value is often safer but in this case I totally understand how it's justified.